bionic/core/task_execution.py

```diff
@@ -59,22 +59,29 @@ def __init__(self, state):
         self.state = state
         self.future = None
         self.results_by_dnode = None
-        self.stage = EntryStage.COMPLETED
+        self._stage = None
 
         # This is initially set to None to avoid eagerly recursing through the entire
         # DAG. We set this once we start processing the entry.
         self.dep_entries = None
 
         self.incoming_reqs = set()
         self.outgoing_reqs = set()
+        self.priority = EntryPriority.DEFAULT
+
+        self.stage = EntryStage.COMPLETED
```

Comment:
This is mostly for my understanding of setters, is there a reason to do it at the end? Like is it supposed to be treated like an initialize method that we would call at the end (once all the variables are initialized)?

Reply:
Oh, no, there's no good reason. This was leftover from when I had some more debugging code in the `stage` property. I'll move it back.